### DISCUSSION TITLE
pipewire <-> pulseaudio

### DISCUSSION THREAD

**Discussion by ekrogh** [microsoft/WSL#11406] [Q&A]
It seems to as if, when you run an up-date/-grade of ubuntu in wsl, you get pipewire + wireplumber installed and then sound stops working.

In my system:
pactl info
Server String: unix:/mnt/wslg/PulseServer
Library Protocol Version: 35
Server Protocol Version: 35
Is Local: yes
Client Index: 0
Tile Size: 65472
User Name: wslg
Host Name: stendell
Server Name: pulseaudio
Server Version: 16.1-8-g6f04
Default Sample Specification: s16le 2ch 44100Hz
Default Channel Map: front-left,front-right
Default Sink: RDPSink
Default Source: RDPSource
Cookie: 4c32:1dde

Is there any way that I can get pipewire to work with the wsl(g) ?  (Example would be best).

Now I disable/mask all *wire* to get sound through.

**Comment by fourpastmidnight** [comment]
On Arch, anyway, there's pipewire-pulse. Is there something similar for Ubuntu?
